README.rst

```diff
@@ -40,7 +40,6 @@ below:
 
 In the future, I plan to support the following extra features:
 
-- `happi <https://github.com/slaclab/happi>`_ will enable device loading from a happi database, and from the experiment questionaire.
```

Comment:
Actually, one more nitpick: this line was originally moved to the top section, and now fully deleted.  It should have been moved into the middle section.

Reply:
👍